react/ui-components/src/hoc/app-config-screen-builder/AppConfigurationWrapper.js

```
@@ -0,0 +1,863 @@
+          onClick={async () => {
+            await handleSubmit();
+          }}
+        />
+      </div>
+      {true && (
+        <SidePanel
+          bgActive
+          className="app-configuration-side-panel"
+          defaultOpenWidth={369}
+          closedContents={[]}
+          closedFooter={[<en />]}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)